plugins/GSdx/GSState.cpp

```diff
@@ -465,6 +465,21 @@ GSVector4i GSState::GetFrameRect(int i)
 	return rectangle;
 }
 
+int GSState::GetFramebufferHeight()
+{
+	const GSVector4i output[2] = { GetFrameRect(0), GetFrameRect(1) };
+	// Framebuffer height is 11 bits max according to GS user manual
+	const int height_limit = (1 << 11) - 1;
+	int top = std::min(output[0].top, output[1].top);
+	int bottom = std::max(output[0].bottom, output[1].bottom);
+	// According to Gregory, GS seems to wrap when values are passed over limit.
```

Comment:
Sorry, you misunderstood me.

GS memory is a flat array. The address wrap after the 4MB.
```
u8 memory[4MB]

address = compute_address_based(base pointer, buffer width, X, Y) % 4MB
```
So X, Y can range from 0 to UINT_MAX. Address will always remain in the memory.